src/integrations/Engage/browser.js

```diff
@@ -3,7 +3,7 @@ import logger from '../../utils/logUtil';
 import { NAME } from './constants';
 import { LOAD_ORIGIN } from '../../utils/ScriptLoader';
 import { refinePayload, getDestinationExternalID } from './utils.js';
-import { getDefinedTraits } from '../../utils/utils';
+import { getDefinedTraits } from '../../utils/commonUtils';
 import { removeUndefinedAndNullValues } from '../../utils/commonUtils';
```

Comment:
We can combine these two lines.

Reply:
There are more such issues even in files not part of the review. We need to fix all the linting issues at once.